anta/device.py

```diff
@@ -323,7 +323,7 @@ async def _collect(self, command: AntaCommand) -> None:
             # No password
             commands.append({"cmd": "enable"})
         if command.revision:
-            commands.append({"cmd": command.command, "revision": command.revision})
+            commands.append({"cmd": command.command, "revision": command.revision})  # type: ignore[dict-item]
```

Comment:
`cli` method of the aioeapi session object expect strings but `command.revision` has to be an `int` to work properly.

```bash
anta/device.py:326: error: Dict entry 1 has incompatible type "str": "int"; expected "str": "str"  [dict-item]
```